bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/common/Price.java

```diff
@@ -19,6 +19,8 @@
  */
 public interface Price {
 
+    boolean isEmpty();
```

Comment:
It could also need some javadoc what empty means and what to expect how the other methods behave when it is empty.